ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -147,55 +147,64 @@ def __init__(
     def get_communicator(worker_id, base_port, timeout_wait):
         return RpcCommunicator(worker_id, base_port, timeout_wait)
 
-    def executable_launcher(self, file_name, docker_training, no_graphics, args):
+    @staticmethod
+    def environment_launch_check(env_path):
+        if not (glob.glob(env_path) or glob.glob(env_path + ".*")):
+            raise UnityEnvironmentException(
```

Comment:
Seems like just returning `None` instead of is more consistent, since you check for None in the calling code.

Reply:
That's a solid point, changing